cql/ManageRareAbnormality.cql

```diff
@@ -673,6 +684,17 @@ define MostRecentCin2BiopsyInPast2Years:
     )
   )
 
+define MostRecentHsilBiopsyInPast6Years:
+    Coalesce(
```

Comment:
I suggest refactoring in future update and leaving it alone for now.

Reply:
That's fair, since removing repetition between the different sections of the management CQL files could turn into a larger task. I opened a Jira ticket that I place at the bottom of the backlog:

**CQL Refactor: Organize and Combine Biopsy Lookbacks**: [CCSMCDS-81](https://jira.mitre.org/browse/CCSMCDS-81)